### ISSUE TITLE
change path /codingchallenges/ to /challenges/

### ISSUE THREAD

**Issue by shiffman** [CodingTrain/website#568]
I would like to keep URLs as short as possible when possible.

I propose changing:

http://codingtra.in/codingchallenges/

to:

http://codingtra.in/challenges/

Will this break anything important? @nilsweb Does Jekyll allow for easily redirecting old links?

**Comment by nilsweb** [comment]
It _shouldn't_ break anything. The only problem is that there's no easy way to set up 301 redirects (at least not that I know of) since those are handled by the server which you can't really configure when using GitHub Pages. Maybe those two articles might be useful:

- https://konradpodgorski.com/blog/2013/10/21/how-i-migrated-my-blog-from-wordpress-to-octopress/#redirect-301-on-github-pages
- https://help.github.com/articles/redirects-on-github-pages/

If anyone knows about an easier/more robust way of doing redirects, please let me know.

**Comment by shiffman** [comment]
Looks like @meiamsome is doing some redirects in (the greatest pull request of all time) #570?

**Comment by nilsweb** [comment]
Yes, he does! Maybe we could introduce the new URL style along with it? Maybe @meiamsome's script could be slightly adjusted to incorporate that as well? I'm trying to review it over the weekend as best as I can since there are A LOT of changes... 😅

**Comment by meiamsome** [comment]
oh right, well I guess all the files will have redirects from their old locations anyway, so we are free to just perform this change simultaneously. Would it just be renaming the folder? If so, I should be able to add that on the end of those changes!

**Comment by nilsweb** [comment]
We would need to rename the folder, all references to coding challenges in the ˋ.mdˋ files and hard-coded links (like in the header).

**Comment by Dhakshith** [comment]
I have forked the repository, cloned it locally in my computer, renamed the folder using git, and pushed it, but i do not know **all instances** of the link. I think i can do a program to find **all** "http://codingtra.in/codingchallenges/" and replace them with "http://codingtra.in/challenges/". I should change them before I can make a pull request.
